src/core/compiler.rkt

```diff
@@ -77,7 +72,7 @@
                    (set! exprc (+ 1 exprc))
                    (set! icache (cons node icache))))))
 
-  (define roots (list->vector (map munge exprs)))
+  (define roots (list->vector (map (compose munge remove-approx) exprs)))
```

Comment:
It’s fine but I bet we could also make this part of munge if we wanted, saving a function name.

Reply:
Simplified